code/modules/economy/account.dm

```diff
@@ -10,7 +9,7 @@
 	/// If TRUE, SSeconomy will store an account into `SSeconomy.bank_accounts`
 	var/add_to_accounts = TRUE
 	var/account_id
-	var/being_dumped = FALSE //pink levels are rising
+	var/crab_protected = NONE //pink levels are rising - used for CRAB-17
```

Comment:
Is there actually a point of using flags here at all? This could be a number instead to indicate how many dumps they are affected by so that when 1 is destroyed, the counter can decrement and they can be blocked if there are still other machines affecing them

Reply:
since you said multiple machines can exist, I changed each machine takes `list/protected_accounts`. Its owner will be siphoned too, but they'll get money back - and this is necessary because someone can meta them if they know where the siphoning machine is with this.